oshu_manor.lic

```diff
@@ -0,0 +1,148 @@
+custom_require.call(%w(common common-travel drinfomon))
+
+class Oshu
+  include DRC
+  include DRCT
+
+  def initialize
+    settings = get_settings
+    @ignored_npcs = settings.ignored_npcs
+    @waiting_room = 2317
+    @graves = [2334, 2337, 2338, 2317]
+    @grexits = [2356, 2349, 2347]
+    Flags.add('allready', "The gargoyle head doesn't budge.")
+    Flags.add('blasted', "The blast catches you fully and sends you careening northward through the air!")
+    Flags.add('head-reset', 'A stone gargoyle head suddenly makes a series of loud grinding')
+    Flags.add('grave_exit', 'You dig to the surface')
+    Flags.add('retreat', 'closes to pole weapon range on you', 'closes to melee range on you', 'begins to advance on you', 'advances from nearby and is closing steadily' )
+    grave_digger
+  end
+  
+  def germfree?
+    return (DRRoom.npcs - @ignored_npcs).empty?

```

Comment:
This was to check for no germish'din (330 to 520) in the room. The player needs to dig grave (8 second rt + 4-5 seconds of wait time) to get to the next area. Its to avoid getting hurt on the way to the hunting zone.